src/components/Popup/Popup.tsx

```diff
@@ -160,41 +148,22 @@ export default class Popup extends UIComponent<Extendable<IPopupProps>, IPopupSt
   private renderPopperChildren = (
     rtl: boolean,
     accessibility: IAccessibilityBehavior,
-    { ref, style }: PopperChildrenProps,
+    { ref, style: popupPlacementStyles, placement }: PopperChildrenProps,
   ) => {
-    const { basic, content } = this.props
-    const computedStyle = rtl ? rtlCSSJS(style) : style
+    const { content } = this.props
 
     return (
-      <Popup.Content
-        innerRef={ref}
-        basic={basic}
-        {...rtl && { dir: 'rtl' }}
-        styles={computedStyle}
-        {...accessibility.attributes.popup}
-        {...accessibility.keyHandlers.popup}
-      >
-        {content}
-      </Popup.Content>
+      <Ref innerRef={domElement => ref(domElement)}>
```

Comment:
we should define a method for this and not define it here every time since it's hurting perf and it's not a good React practice

Reply:
have replied on that in the first comment about arrow functions